### DISCUSSION TITLE
Recent releases to sam cli causing breaking changes for sync feature

### DISCUSSION THREAD

**Discussion by cnguyenrettig** [aws/aws-sam-cli#5044] [Questions]
Recent releases to sam cli above version 1.74 have caused the sync serverless application feature to error out. 

The workaround was to downgrade to 1.74, however the latest version of AWS toolkit (1.65-223) requires sam cli version 1.78 or higher. 

Has anyone else encountered this and found a workaround other than downgrading everything to previous versions. 

This is the last combination that worked:

1. AWS Toolkit plugin: 1.62 or below*
2. Sam Cli: 1.74 
*Use IntelliJ Plugins window to uninstall and then install the previous AWS Toolkit from disk if you have upgraded to 1.64 or higher.

**Comment by super132** [comment]
Thanks for reaching out to us. May I know what's the error that you encountered while using `sam sync` command in SAM CLI version above 1.74?
